app/java/Procfile

```diff
@@ -1,2 +1,2 @@
-web: java -jar target/cloud-client-api-0.0.0-SNAPSHOT.jar
-process: java -jar target/cloud-client-api-0.0.0-SNAPSHOT-processingjob.jar
+web: java -jar target/cloud-client-api-0.0.0-SNAPSHOT.jar -Dspring-boot.run.profiles=default
+process: java -jar target/cloud-client-api-0.0.0-SNAPSHOT-processingjob.jar -Dspring-boot.run.profiles=default
```

Comment:
To make the data `process`-ing job output logs locally (instead of Cloud Logging), we need to replace `default` with `locallogging`:
```
java -jar target/cloud-client-api-0.0.0-SNAPSHOT-processingjob.jar -Dspring-boot.run.profiles=locallogging
```

However, this is not documented anywhere currently.
@glasnt, what are you thoughts on introducing a `make` command for local execution of the `process` job? Alternatively, I can document the above state here: https://github.com/GoogleCloudPlatform/terraform-cloud-client-api/tree/main/app#application

Reply:
Having a `make process` that is for local execution of the job makes sense. It would be the companion to `make dev`. I'll create an issue/PR to add this. (This new `make` target shouldn't block this PR, it's a development helper only.)

Reply:
I just approved https://github.com/GoogleCloudPlatform/terraform-cloud-client-api/pull/92.